### DISCUSSION TITLE
[Feat]: Problems in use

### DISCUSSION THREAD

**Discussion by chenhuanguang** [coder/code-server#5677] [Q&A]
The command to start the code server is ‘nohup ./bin/code-server --port 8888 --host 0.0.0.0 --ignore-last-opened --auth none’
How should I close it correctly

**Comment by jsjoeio** [comment]
I don't use `nohup` but seems like you need to kill the process with `kill <pid>`

See here: https://stackoverflow.com/questions/8007380/how-to-kill-a-nohup-process
